<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gui</class>
 <widget class="QMainWindow" name="gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>神龙工具箱</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="adb_backup">
       <attribute name="title">
        <string>安卓应用备份</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_1">
          <item>
           <widget class="QPushButton" name="aab_btn_device">
            <property name="maximumSize">
             <size>
              <width>140</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>刷新连接设备</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="aab_cmb_device"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="aab_btn_path">
            <property name="maximumSize">
             <size>
              <width>140</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>选择备份文件路径</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="aab_label_path">
            <property name="text">
             <string/>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="2">
           <widget class="QPushButton" name="aab_btn_clear">
            <property name="text">
             <string>清空日志</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QCheckBox" name="aab_ckb_apk">
            <property name="text">
             <string>包括安装包</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="aab_btn_getapp_high">
            <property name="text">
             <string>获取包名(高版本)</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="aab_btn_backup">
            <property name="text">
             <string>备份</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="aab_btn_getapp_low">
            <property name="text">
             <string>获取包名(低版本)</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="aab_btn_restore">
            <property name="text">
             <string>恢复</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="aab_label_name">
            <property name="text">
             <string>软件包名：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="aab_lineedit_name"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="aab_textedit_log">
          <property name="font">
           <font>
            <family>FiraCode Nerd Font</family>
           </font>
          </property>
          <property name="plainText">
           <string>神龙工具箱 v1.0
作者：abcdesteve
    _                                   _   ____   ___ ____  _____ 
   / \__   _____   ___ ___  _ __   __ _| | |___ \ / _ \___ \|___ / 
  / _ \ \ / / _ \ / __/ _ \| '_ \ / _` | |   __) | | | |__) | |_ \ 
 / ___ \ V / (_) | (_| (_) | | | | (_| | |  / __/| |_| / __/ ___) |
/_/   \_\_/ \___/ \___\___/|_| |_|\__,_|_| |_____|\___/_____|____/ 
   _   _ _       _      _   _                                       _ 
  /_\ | | |  _ _(_)__ _| |_| |_ ___  _ _ ___ ___ ___ _ ___ _____ __| |
 / _ \| | | | '_| / _` | ' \  _(_-&lt; | '_/ -_|_-&lt;/ -_) '_\ V / -_) _` |
/_/ \_\_|_| |_| |_\__, |_||_\__/__/ |_| \___/__/\___|_|  \_/\___\__,_|
                  |___/                                               </string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="link">
       <attribute name="title">
        <string>创建符号链接</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="5" column="2">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>226</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="6" column="1">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="5" column="1">
         <widget class="QPushButton" name="csl_btn_start">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>创建</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="3">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="csl_btn_from">
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>选择本体路径</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="csl_lineedit_from">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="placeholderText">
             <string>请先选择本体路径</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="5" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>226</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="0" colspan="3">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QPushButton" name="csl_btn_to">
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>选择目标路径</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="csl_lineedit_to">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="placeholderText">
             <string>请先选择目标路径(确保该路径不存在)</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="1">
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
